README.md

```diff
@@ -491,7 +491,7 @@ $ docker run --name my-mongo -d mongo
 $ docker run --name my-parse-server --link my-mongo:mongo -v config-vol:/parse-server/config -p 1337:1337 -d parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://mongo/test --publicServerURL http://localhost:1337/parse --mountGraphQL --mountPlayground
 ```
 
-***Note:*** *If you want to use [Cloud Code](https://docs.parseplatform.org/cloudcode/guide/) feature, please add `-v cloud-code-vol:/parse-server/cloud --cloud /parse-server/cloud/main.js` to command above. Make sure the `main.js` file is available in the `cloud-code-vol` directory before run this command. Otherwise, an error will occur.*
+***Note:*** *If you want to use [Cloud Code](https://docs.parseplatform.org/cloudcode/guide/) feature, please add `-v cloud-code-vol:/parse-server/cloud --cloud /parse-server/cloud/main.js` to command above. Make sure the `main.js` file is available in the `cloud-code-vol` directory before running this command. Otherwise, an error will occur.*
```

Comment:
This information seems to occur twice in the docs. Can this be removed and if necessary just mention a reference to the other section?

Reply:
I think one is related to Docker and the other GraphQL

Reply:
I see, well, this PR is a "minor spelling improvement", so maybe another time, but that duplicate info should be extracted and only referenced two times.

Reply:
Yeah, I think this PR is only somewhat temporary. @TomWFox and I are still working on the complete overhaul of the README